test/integration-test/PassPreprocessor/VarDeclarationsTest.cpp

```diff
@@ -0,0 +1,56 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                         _       _
+//                        | |     | |
+//                    __ _| |_ ___| | __ _ _ __   __ _
+//                   / _` | __/ __| |/ _` | '_ \ / _` |
+//                  | (_| | || (__| | (_| | | | | (_| |
+//                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
+//                    __/ |                       __/ |
+//                   |___/                       |___/
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+// RUN: %gtclang% %file% -fno-codegen -freport-pass-preprocessor
+
+
+#include "gridtools/clang_dsl.hpp"
+
+using namespace gridtools::clang;
+
+stencil_function bar{
+    storage a,b;
+    void Do(){
+        var d = 10;
+        a = b + d;
+    }
+};
+
+stencil Test01 {
```

Comment:
Good test, but aren't you missing the checks? ;)

Reply:
this checks if compilation works. If we'd actually implement the checks, they would be at very strange lines since we add lines to the preprocessed file. Is it worth it to have an "ignore number" command or is it fine that they are in odd places?